### DISCUSSION TITLE
0.8.1 - db updates gone after docker restart?

### DISCUSSION THREAD

**Discussion by Trashmee** [blakeblackshear/frigate#723] [Ask A Question]
Hi,

with 0.8.1-amd64 my database doesn't seem to get the updates it used to get - is that due to my config?
My frigate.db doesn't seem to get updated everytime that events occur.

That means everything is fine & displays well until I restart the container. In that case the references are gone.

What's the purpose of the files frigate.db-shm / -wal - and should i include them in docker-compose?

docker-compose.yaml
```
      - frigateclips:/media/frigate/clips
      - frigaterecordings:/media/frigate/recordings
      - <localssd>/frigate.db:/media/frigate/frigate.db
```

```
ll /media/frigate/
drwxrwxrwx 2 root root   311296 Feb  5 09:02 clips/
-rwxrwxrwx 1 root root 23412736 Feb  4 15:32 frigate.db*
-rwxrwxrwx 1 root root    32768 Feb  5 09:02 frigate.db-shm*
-rwxrwxrwx 1 root root   444992 Feb  5 09:02 frigate.db-wal*
drwxrwxrwx 2 root root     4096 Dec 23 16:06 recordings/
```

Kind regards, 
Alex

**Comment by blakeblackshear** [comment]
It depends on whether or not the processes exit cleanly. I would recommend mounting a directory at `/media/frigate/data` and using `/media/frigate/data/frigate.db` as the path in your config.

Unfortunately, once the database is lost, frigate no longer knows about the existing files. They will need to be manually cleaned up when you no longer want them.

**Comment by Trashmee** [comment]
That worked out, thanks!
